styles/style.css

```diff
@@ -6,3 +6,265 @@ Green: #00B172
 White: #FFF
 
 */
+/* 
+===================================================
+CLEARING BROWSER DEFAULTS AND SETTING GLOBAL STYLES
+===================================================
+ */
+body {
+  margin: 0;
+  font-size: 1.5rem;
+  font-weight: lighter;
+  color: #1a1a1a;
+}
+
+/* 
+==========================================
+LAYOUT
+==========================================
+ */
+
+section,
+header {
+  border: 1px red solid;
+}
+.row,
+col {
+  border: 1px green dashed;
+}
+/* layout guides, to be used in development only */
+nav {
+  width: 100%;
+  height: 4.5rem;
+  background-color: #fff;
+}
+.logo {
+  width: 10%;
+  margin: 1rem 0 0 1.5rem;
+}
+.hamburger {
+  display: none;
+  padding: 15px;
+  cursor: pointer;
+  #mobile_menu {
+    color: #00b172;
+  }
+}
+.off {
+  display: none;
+}
+.on {
+  display: block;
+  background-color: #fff;
+  padding-bottom: 0.5rem;
+  margin: 0 0 0 0;
+}
+ul {
+  list-style-type: none;
+  display: flex;
+  float: right;
+}
+li {
+  font-size: 1.5rem;
+  margin: 0.5rem 2rem 0 0;
+  font-weight: light;
+}
+
+.landing {
+  min-height: 30rem;
+  background-image: url("../images/landing.jpg");
+  background-size: contain;
+}
+.landing span {
+  position: absolute;
+  top: 40%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: #fff;
+  text-align: center;
+}
+.landing h1 {
+  font-size: 3rem;
+}
+.landing p {
+  font-weight: 100;
+}
+
+.features {
+  min-height: 25rem;
+  margin: 2rem 0 2rem 0;
+}
+.features h2 {
+  text-align: center;
+  text-decoration: underline #00b172;
+  text-underline-position: under;
+  margin-bottom: 2em;
+}
+.features h3 {
+  color: #00b172;
+}
+.features .col-1 {
+  text-align: center;
+}
+.features article {
+  margin: 1rem 0 2rem 0;
+}
+
+.features .row article img {
+  width: 20%;
+}
+.features .row {
+  justify-content: space-evenly;
+  margin: 0 0 3rem 0;
+}
+
+.app {
+  width: 95%;
+  background-color: #00b172;
+  margin: 0 auto;
+  color: #fff;
+}
+.app main {
+  margin: 1rem 1rem 1rem 1rem;
+}
+.app h2 {
+  text-decoration: underline #fff;
+  text-underline-position: under;
+}
+.app article {
+  margin: 2rem 0 0 0;
+}
+.app .row {
+  justify-content: space-evenly;
+}
+.app main {
+  margin: 1.5rem;
+}
+.app span {
+  display: block;
+  width: 80%;
+}
+.app img {
+  width: 100%;
+  max-height: 100%;
+}
+.app .col-3 {
+  align-content: center;
+}
+.app .col-1 img {
+  margin-top: 5rem;
+  width: 95%;
+}
+
+section {
+  width: 100%;
+}
+.row {
+  display: flex;
+}
+.col-1 {
+  width: 25%;
+}
+.col-2 {
+  width: 45%;
+}
+.col-3 {
+  width: 10%;
+}
+.col-4 {
+  width: 30%;
+}
+
+/* set styles for all other devices such as desktopts, larger than mobiles phones*/
+@media (max-width: 645px) {
+  section {
+    width: 100%;
+  }
+  .row {
+    display: flex;
+    flex-direction: column;
+  }
+  .col-1 {
+    width: 100%;
+  }
+  .col-2 {
+    width: 100%;
+  }
+  .col-3 {
+    width: 100%;
+  }
+  .col-4 {
+    width: 100%;
+  }
```

Comment:
Since they have the same rule, can be added all at once:
```
.col-1, .col-2, .col-3, .col-4 {
    width: 100%;
}
```

Reply:
Oh yh
I actually know this.
I don't know why I didn't implement it.
Thanks